notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,714 @@
+    "```\n",
+    "  instance_group [\n",
+    "    {\n",
+    "      count: 2\n",
+    "      kind: KIND_GPU\n",
+    "    }\n",
+    "  ]\n",
+    "```\n",
+    "This will result in Triton server creating 2 instances of XGBoost model on the same GPU. Run the server after making the above modification:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Writing config to file\n",
+    "cat > /data/model_repository/fil/config.pbtxt <<EOL \n",
+    "name: \"fil\"                              # Name of the model directory (fil in our case)\n",
+    "backend: \"fil\"                           # Triton FIL backend for deploying forest models\n",
+    "max_batch_size: 8192\n",
+    "input [\n",
+    " {\n",
+    "    name: \"input__0\"\n",
+    "    data_type: TYPE_FP32\n",
+    "    dims: [ 9 ]                          # Input feature dimensions, in our sample case it's 9\n",
+    "  }\n",
+    "]\n",
+    "output [\n",
+    " {\n",
+    "    name: \"output__0\"\n",
+    "    data_type: TYPE_FP32\n",
+    "    dims: [ 2 ]                          # Output 2 for binary classification model\n",
+    "  }\n",
+    "]\n",
+    "instance_group [                         # 2 instances of the model will be deployed in this case\n",
+    "    {\n",
+    "      count: 2\n",
+    "      kind: KIND_GPU\n",
+    "    }\n",
+    "  ]\n",
+    "parameters [\n",
+    "  {\n",
+    "    key: \"model_type\"\n",
+    "    value: { string_value: \"xgboost\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"predict_proba\"\n",
+    "    value: { string_value: \"false\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"output_class\"\n",
+    "    value: { string_value: \"true\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"threshold\"\n",
+    "    value: { string_value: \"0.5\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"algo\"\n",
+    "    value: { string_value: \"ALGO_AUTO\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"storage_type\"\n",
+    "    value: { string_value: \"AUTO\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"blocks_per_sm\"\n",
+    "    value: { string_value: \"0\" }\n",
+    "  }\n",
+    "]\n",
+    "\n",
+    "dynamic_batching {\n",
+    "  preferred_batch_size: [1024, 8192]\n",
+    "  max_queue_delay_microseconds: 100\n",
+    "}\n",
+    "EOL"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run the Triton Inference Server with 2 instances of our model in a Subprocess from Jupyter notebook\n",
+    "\n",
+    "# The os.setsid() is passed in the argument preexec_fn so\n",
+    "# it's run after the fork() and before  exec() to run the shell.\n",
+    "pro = subprocess.Popen([\"tritonserver --model-repository=/data/model_repository\"], stdout=subprocess.PIPE, \n",
+    "                       shell=True, preexec_fn=os.setsid) "
```

Comment:
Another instance of the `pro` variable that could be a bit longer to be more descriptive and look nicer